aea/test_tools/acn_image.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""ACN Docker Image."""
+
+import logging
+import socket
+import time
+from typing import Dict, List
+
+from aea.exceptions import enforce
+from aea.test_tools.docker_image import Container, DockerClient, DockerImage
+
+
+META_ADDRESS = "0.0.0.0"  # nosec
+
+# created agent: bootstrap_peer
+#     private key: 7f669ab5eee5719e385f7aeb1973769fc75b7cbbe0850ca16c4eabe84e01afbd
+#     public key:  0270475f9b78c0285a6ac6067582f5e159ec147ccb03aee16a32731f68920b1ae8
+#     PeerID:      16Uiu2HAm2yxmLQTZTrxjo5c4k5ka8AVMcpeD5zMMeasE6xDw1YQw
+#
+# created agent: entry_node_1
+#     private key: 5b8d3be9f27489040a01adc2b746b9a5f9d32ed843d99cf7d2995c8140636190
+#     public key:  02197b55d736bd242311aaabb485f9db40881349873bb13e8b60c8a130ecb341d8
+#     PeerID:      16Uiu2HAkw99FW2GKb2qs24eLgfXSSUjke1teDaV9km63Fv3UGdnF
+#
+# created agent: entry_node_2
+#     private key: cc096b7be575c11d3d3d2f8a9c9be9bd59b351317b2a114ef7e014cc5a92508e
+#     public key:  0287ee61e8f939aeaa69bd7156463d698f8e74a3e1d5dd20cce997970f13ad4f12
+#     PeerID:      16Uiu2HAm4aHr1iKR323tca8Zu8hKStEEVwGkE2gtCJw49S3gbuVj
```

Comment:
do we need these here?

Reply:
nope this can go. Will do that in the next PR where I introduce a bit of documentation containing bash script snippets explaining how to generate these keys (which is not at open-acn and will be moved here)